src/WebJobs.Script.WebHost/Diagnostics/ExtendedEventSource.cs

```diff
@@ -242,5 +242,35 @@ protected void WriteEvent(int eventNum, string a, string b, string c, string d,
                 WriteEventCore(eventNum, count, data);
             }
         }
+
+        // RaiseFunctionsDiagnostic
+        [NonEvent]
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        protected void WriteEvent(int eventNum, string a, string b, string c, string d, string e, string f)
+        {
+            int count = MethodBase.GetCurrentMethod().GetParameters().Length;
```

Comment:
Can you please change looking up count using MethodBase rest of the places in this file?